events/reporter.go

```diff
@@ -118,7 +118,7 @@ func ReportRewardReceived(r Reward) {
 // ReportNewBlock reports a new block
 func ReportNewBlock(blk *types.Block) {
 	Publish(NewBlock{
-		ID:    blk.ID().String(),
+		ID:    blk.ID().ShortString(),
```

Comment:
While I think the code here does the right thing, it's a bit misleading because at a glance `ShortString` sounds like it abbreviates the block ID. We might want to make this more explicit.

Reply:
It DOES return the abbreviated version. If this is not what we want we should change it back to use `.String()` (which used to return the shortened version, but now does what it says).

Reply:
I just realized that this is for the API... I said in another comment that `.String()` was not used in the API and I was wrong. Why do we use strings here at all? Why not bytes directly? That's what we do here:

https://github.com/spacemeshos/go-spacemesh/blob/f76c8b99427a4a9f559da87fced6cf5aa3219266/api/grpcserver/mesh_service.go#L320-L323

Reply:
ok, so for the record - this is NOT for the API - it's for pubsub. Leaving it as is.